### DISCUSSION TITLE
Error - pair_style command not found

### DISCUSSION THREAD

**Discussion by AnuragKr** [deepmodeling/deepmd-kit#1845] [Q&A] [answered]
Hello,

I have installed deepmd via conda. After activating the environment, I have also set the path in the bashrc file as mentioned in  [#1792](https://github.com/deepmodeling/deepmd-kit/discussions/1792) still not recognizing the command.
I am doubtful about how to set a path when you are running via  conda environment.

1. (deepmd) anurag1@hp-HP-Z8-G4-Workstation:~/.local/deepmd-kit/examples/water/se_e2_a$ echo $LAMMPS_PLUGIN_PATH
/home/anurag1/miniconda3/envs/deepmd/lib/deepmd_lmp:

2. lmp -h
more: /home/anurag1/miniconda3/envs/deepmd/lib/libtinfo.so.6: no version information available (required by more)

Large-scale Atomic/Molecular Massively Parallel Simulator - 23 Jun 2022

Usage example: lmp -var t 300 -echo screen -in in.alloy

List of command line options supported by this LAMMPS executable:

-echo none/screen/log/both  : echoing of input script (-e)
-help                       : print this help message (-h)
-in none/filename           : read input from file or stdin (default) (-i)
-kokkos on/off ...          : turn KOKKOS mode on or off (-k)
-log none/filename          : where to send log output (-l)
-mdi '<mdi flags>'          : pass flags to the MolSSI Driver Interface
-mpicolor color             : which exe in a multi-exe mpirun cmd (-m)
-cite                       : select citation reminder style (-c)
-nocite                     : disable citation reminder (-nc)
-package style ...          : invoke package command (-pk)
-partition size1 size2 ...  : assign partition sizes (-p)
-plog basename              : basename for partition logs (-pl)
-pscreen basename           : basename for partition screens (-ps)
-restart2data rfile dfile ... : convert restart to data file (-r2data)
-restart2dump rfile dgroup dstyle dfile ... 
                            : convert restart to dump file (-r2dump)
-reorder topology-specs     : processor reordering (-r)
-screen none/filename       : where to send screen output (-sc)
-skiprun                    : skip loops in run and minimize (-sr)
-suffix gpu/intel/opt/omp   : style suffix to apply (-sf)
-var varname value          : set index style variable (-v)

OS: Linux "Ubuntu 20.04.4 LTS" 5.13.0-52-generic x86_64

Compiler: GNU C++ 7.5.0 with OpenMP 4.5
C++ standard: C++11
MPI v3.1: MPICH Version:	3.3.2
MPICH Release date:	Tue Nov 12 21:23:16 CST 2019
MPICH ABI:	13:8:1
Accelerator configuration:

Thanks.

**Comment by wanghan-iapcm** [comment]
@njzjz could you please take care?

**Comment by njzjz** [comment]
Could you provide the output of `conda list`?

**Reply by AnuragKr** [reply]
![Screenshot 2022-08-06 at 3 07 35 PM](https://user-images.githubusercontent.com/10074710/183243539-5a84f638-09b3-4f1b-b2de-bf58c8a13a53.png)
![Screenshot 2022-08-06 at 3 08 14 PM](https://user-images.githubusercontent.com/10074710/183243554-d8bfb549-39df-4d12-b92b-1a493778cfa9.png)
![Screenshot 2022-08-06 at 3 08 35 PM](https://user-images.githubusercontent.com/10074710/183243567-96a46262-a04d-48bf-98ad-b979aab242dd.png)
![Screenshot 2022-08-06 at 3 08 55 PM](https://user-images.githubusercontent.com/10074710/183243578-f08ee2df-013e-497c-b05d-ae59c2e461f2.png)

**Reply by njzjz** [reply]
Here is nothing wrong. Is there any output shown by [`plugin list`](https://docs.lammps.org/plugin.html)?

**Reply by AnuragKr** [reply]
![Screenshot 2022-08-07 at 6 08 20 PM](https://user-images.githubusercontent.com/10074710/183290893-c110a176-ab04-4dae-9540-a7d828831971.png)

**Reply by njzjz** [reply]
This is LAMMPS command instead of shell, which should be executed in LAMMPS.

**Reply by AnuragKr** [reply]
Could you please tell How to start LAMMPS shell and execute that command.I am unable to find anything related to plugin list Lammps command.

**Reply by njzjz** [reply]
Typing `lmp` in the shell.

**Reply by AnuragKr** [reply]
Thanks for the answer.

![lammps_plugin](https://user-images.githubusercontent.com/10074710/183565999-f2c5c118-d280-4816-9ee2-167a6851fbfa.png)

**Reply by njzjz** [reply]
It looks good though.

**Comment by AnuragKr** [comment, marked as answer]
Thanks for the response. I have solved the issue as I was doing it wrongly. Now I am able to run LAMMPS simulation.
